### DISCUSSION TITLE
Checkout to a specific vcpkg tag

### DISCUSSION THREAD

**Discussion by gvnnz** [actions/virtual-environments#5585] [General questions] [answered]
Currently most of the virtual envs provide vcpkg 2022 (build from master <0cf5b4305>). I need to checkout to a specific vcpkg tag instead, in order to pin the package versions provided by vcpkg. I know I could clone the vcpkg repo from scratch and configure it manually, but it would be less time-consuming to use what's already available in the virtual environment. Is there an easy way to do that?

**Comment by miketimofeev** [comment, marked as answer]
@gvnnz we clone full vcpkg repo here
https://github.com/actions/virtual-environments/blob/e4d58452f4d4fc085b24c7be9bb6d57a3ba4d3ec/images/win/scripts/Installers/Install-Vcpkg.ps1#L10
So you can simply go to `C:\vcpkg` directory and checkout the commit you need

**Reply by gvnnz** [reply]
Great. Thanks a lot @miketimofeev
